src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -242,10 +242,6 @@ private TestSuite BuildTestAssembly(Assembly assembly, string assemblyName, ILis
 #endif
             testAssembly.Properties.Set(PropertyNames.AppDomain, AppDomain.CurrentDomain.FriendlyName);
 #endif
-
-            // TODO: Make this an option? Add Option to sort assemblies as well?
-            testAssembly.Sort();
-
```

Comment:
This sort should remain as is, until we change it in a separate issue. It is not intended to provide execution order, but to control the display order of tests in the gui.